### DISCUSSION TITLE
Possible enhancements according to the feedback of paleoHackathon

### DISCUSSION THREAD

**Discussion by fzhu2e** [LinkedEarth/Pyleoclim_util#144] [Ideas]
1. return the step, start, end information in `gkernel()` for users to access
2. add label options in `to_LipdSeries()`
3. allow `series_dict` as an alternative input to `series_list` for `MutilpleSeries` construction, so that the keys will be used as labels for each `Series`.
4. Have you thought about potential of spatial scales in place of time scales? (from Jordan Landers)

**Comment by CommonClimate** [comment]
5. add a `view()` method for LiPD objects that allows to peruse the contents. Should re-use the ontology structure as much as possible.

**Comment by khider** [comment]
Subdivide MS into Series easily.
